src/org/usfirst/frc4904/robot/commands/PistonIntakeRelease.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsytems.PistonIntake;
+import org.usfirst.frc4904.standard.commands.SingleOp;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class PistonIntakeRelease extends CommandGroup{
```

Comment:
How is this different from rollout?

Reply:
Release just sets the cube down with the pistons, and rollout spins the motors to "spit it out."